### DISCUSSION TITLE
nvidia country not recognised error

### DISCUSSION THREAD

**Discussion by cashewnut21** [jef/streetmerchant#1573] [Help]
Hi i get the following error: 

 "error :: ✖ [nvidia] Unknown country great_briain"

and hence i believe the bot does not search the 'nvidia' nor 'scan' websites (the website where FEs drop in the UK) ive tried "nvidia, nvidia-api,nvidia-snatcher' and none seem to resolve the error!

Thanks !

**Comment by mittal254** [comment]
looks like you spelt 'britain' wrong
